src/couch/src/couch_passwords.erl

```diff
@@ -51,7 +51,10 @@ hash_admin_password("pbkdf2", ClearPassword) ->
                                         Salt ,list_to_integer(Iterations)),
     ?l2b("-pbkdf2-" ++ ?b2l(DerivedKey) ++ ","
         ++ ?b2l(Salt) ++ ","
-        ++ Iterations).
+        ++ Iterations);
+hash_admin_password("bcrypt", ClearPassword) ->
+    LogRounds = list_to_integer(config:get("couch_httpd_auth", "log_rounds", "10")),
```

Comment:
Maybe try `config:get_integer("couch_httpd_auth", "log_rounds", 10)` here. Right above pbkdf2 doesn't do it but since we have `get_integer/3` now it looks a bit cleaner.

Also 10 might look better as a define at the top of the module. Though again, pbkdf2's default value isn't one, so up to you.